### PR TITLE
add blog/2020-09-29-mapscanner

### PR DESCRIPTION
**Pull request by mpadge**

### CODE REVIEW

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+neighboring or partially-enclosing "Hackey" will return a polygonal boundary.)
+Nevertheless, those for whom Dalston is a part of their everyday life know
+where Dalston is. The problem is that each person has a slightly different
+idea. My own delineation of Dalston might look something like the following:
+
+<!--
+{{< figure src = "dalston.png" width = "150" alt = "Subjective boundary of Dalston, UK" caption = "Subjective boundary of Dalston, UK">}}
+-->
+
+![](dalston-bdry.png)
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:211]:
```suggestion

```

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  - spatial
+  - maps
+  - survey
+# The summary below will be used by e.g. Twitter cards
+description: "The mapscanner package enables lines drawn by hand on maps to be converted to spatial objects"
+# If you have no preferred image for Twitter cards, 
+# delete the twitterImg line below
+# Note there is no '/' symbol before 'img' here
+# if needed replace blog with technotes
+twitterImg: blog/2020/09/29/mapscanner/poly-aggr-plot.png
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:26]: Update/remove the image for the twitter card. __*Or*__, if you have time, add this image to the bottom of the second use case (see my comments there), and then this is fine :grin:

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+converted to a spatially rectified polygon via the [`ms_rectify_map()`
+function](https://docs.ropensci.org/mapscanner/reference/ms_rectify_map.html).
+
+```{r rectify-dalston}
+library (mapscanner)
+x <- ms_rectify_map (map_original = "dalston.png",      # name of original map
+                     map_modified = "dalston-bdry.png") # scanned version shown above
+x
+```
+
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:223]: __*If*__ you have time, perhaps include an image with the nice polygon?

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+location of this area. In their case, their main result was simply represented
+as a series of overlaid lines from which a central tendency was able to be
+visually perceived. The [mapscanner
+package](https://github.com/ropensci/mapscanner) enables more powerful analyses
+through the
+[`ms_aggregate_polys()`](https://docs.ropensci.org/mapscanner/reference/ms_aggregate_polys.html)
+function which aggregates multiple polygons into a single "multipolygon" object
+in which each component corresponds to the overlap of some number of individual
+polygons.
+
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:237]: __*If*__ you have time, perhaps include that pretty image with all the stacked polygons as an example here? I don't think you need to show any code, the text/caption would explain it well enough. That way you can keep the twitter card :grin:

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+featuring part of the road traversed for that study.) The [mapscanner
+package](https://github.com/ropensci/mapscanner) can be used to analyse
+collective subjective perspectives of space.
+
+Consider, for example, Dalston. Dalston is not, nor has ever been, an official
+administrative location in London, and there are therefore [no official
+boundaries](https://en.wikipedia.org/wiki/Dalston). (This has the effect that
+typing "Dalston" into your preferred online map service will generally return a
+single point, whereas typing in officially-recognized locations like
+neighboring or partially-enclosing "Hackey" will return a polygonal boundary.)
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:202]: Should this be Hack**n**ey?
```suggestion
neighboring or partially-enclosing "Hackney" will return a polygonal boundary.)
```

**Review by steffilazerte — COMMENTED** [review]
Hi @mpadge 
I love the new organization, it really flows well! I have just a couple more comments (sorry!). The only important ones are a potential typo (should it be Hackney rather than Hackey?), a fix to the image (it doesn't show up currently), and dealing with the twitter card. The other suggestions are really only if you a) like them, and b) have time!

I'm one of those people who can't help but make suggestions, so please do not feel like you have to address them :smile_cat: 

Thank you for all your attention to detail throughout this blog post!

**Comment by maelle** [comment]
Shouldn't the h3 be h2?

**Comment by steffilazerte** [comment]
I think it depends, the first heading is ## so the others could be ## or ### depending on what @mpadge wants the substructure of the document to look like (?)

**Comment by maelle** [comment]
ah ok I had missed the h2, all good! it's the first roweb3 PR so I was curious to see how it looked, I should have looked more closely.

**Comment by mpadge** [comment]
>  ... it's the first roweb3 PR so I was curious to see how it looked

so deeply and profoundly honoured i am!!! Thank you all!

**Comment by stefaniebutland** [comment]
What a rich post - love the citizen science aspects !!
Love that Vancouver trees db. 

noticed duplicate text 
> were particularly invaluable in the development were particularly invaluable in the development 

So cool that you have the first PR to roweb3

**Comment by mpadge** [comment]
And as the first PR-ifier, let me attest that the preview site looks soooooo beautiful!! You have all, and obviously very particularly and especially @maelle, done a truly wondrous job. Thank you!

**Comment by steffilazerte** [comment]
> And as the first PR-ifier, let me attest that the preview site looks soooooo beautiful!! You have all, and obviously very particularly and especially @maelle, done a truly wondrous job. Thank you!

Agreed! :tada:

**Pull request merged**